app/controllers/posts_controller.rb

```diff
@@ -62,6 +62,9 @@ def destroy
     # Use callbacks to share common setup or constraints between actions.
     def set_post
       @post = Post.find(params[:id])
+    rescue ActiveRecord::RecordNotFound
+      flash[:notice] = "we couldn't find that post"
```

Comment:
Nit: It's a bit strange to see an error message under green text. Many Rails apps use `flash[:alert]` or `flash[:error]`, but you'd need to update the view to render this (probably near where the `notice` is rendered).

It's good to have error handling, though. 👍🏼